ibis/omniscidb/ddl.py

```diff
@@ -340,6 +341,83 @@ def compile(self):
         return self._wrap_command(action)
 
 
+class AddColumn(AlterTable):
+    """Add Column class."""
+
+    def __init__(self, table_name, **columns_with_types):
+        super().__init__(table_name)
+        self.dict_cols_with_types = {}
+        for key, value in columns_with_types.items():
+            self.dict_cols_with_types[key] = value
+
+    def _pieces(self):
+        col_count = len(self.dict_cols_with_types)
+        if col_count == 0:
+            raise IbisInputError('No column requested to add.')
+        elif col_count == 1:
+            column_name = list(self.dict_cols_with_types.keys())[-1]
+            yield '{} ADD COLUMN {} {};'.format(
+                self.table, column_name, self.dict_cols_with_types[column_name]
+            )
+        else:
+            yield '{} ADD ('.format(self.table)
+            iteration_idx = 0
+            for col, d_type in self.dict_cols_with_types.items():
+                if iteration_idx != col_count - 1:
+                    yield '{} {}, '.format(col, d_type)
+                else:
+                    yield '{} {});'.format(col, d_type)
+                iteration_idx += 1
```

Comment:
@YarShev maybe you can optmize your code using:

```
        if col_count == 0:
            raise IbisInputError('No column requested to add.')
        
        sep = ''
        yield '{} ADD ('.format(self.table)
        for col, d_type in self.dict_cols_with_types.items():
            yield '{} {} {}'.format(sep, col, d_type)
            sep = ','
        yield ');'
```

Reply:
Thanks, done